sophia/src/term/iri.rs

```diff
@@ -375,7 +417,7 @@ impl Iri<&'static str> {
     /// # Safety
```

Comment:
Should this section be renamed, since the function is not `unsafe` anymore?

Reply:
Maybe, it's no longer `unsafe` but still a thread to data consistency. What would you prefer? Warning, Contract, Note, ...?

Reply:
As elsewhere, I suggest `Pre-conditions`